R/wrapers.R

```diff
@@ -472,6 +472,16 @@ torch_nonzero <- function(self, as_list = FALSE) {
   }
 }
 
+#' @rdname torch_where
+torch_where <- function(condition, self = NULL, other = NULL) {
+  if (is.null(self) && is.null(other)) {
+    out <- torch_nonzero(condition, as_list = TRUE)
```

Comment:
I don't think it's a good idea to use a different operation here, unless this is how it's implemented in Python too. This would make it harder to debug in cases like tracing where only the operator name is recorded.

Reply:
I see, thanks! Pushing the updated version in a sec.
However, 

unless this is how it's implemented in Python too. 

https://github.com/pytorch/pytorch/blob/c780610f2d8358297cb4e4460692d496e124d64d/aten/src/ATen/native/TensorCompare.cpp#L132

I guess you could say it is, in a way (although that's not Python ;-))